docs/scenes/get-started/start-developing/ProjectStructure/files/package-json.mdx

```diff
@@ -0,0 +1,5 @@
+import { RawH3 } from '~/ui/components/Text';
+
+<RawH3>package.json</RawH3>
+
+The **package.json** file contains the project's dependencies, scripts and metadata. Anytime a new dependency is added to your project, it will be added to this file.
```

Comment:
```suggestion
The **package.json** file contains the project's dependencies, scripts, and metadata. Anytime a new dependency is added to your project, it will be added to this file.
```